cpp/pmft/PMFTXYT.cc

```diff
@@ -49,7 +49,7 @@ PMFTXYT::PMFTXYT(float x_max, float y_max, unsigned int n_x, unsigned int n_y, u
     m_pcf_array.prepare({n_x, n_y, n_t});
 
     // Construct the Histogram object that will be used to keep track of counts of bond distances found.
-    BondHistogram::Axes axes;
+    util::Axes axes;
     axes.push_back(std::make_shared<util::RegularAxis>(n_x, -x_max, x_max));
     axes.push_back(std::make_shared<util::RegularAxis>(n_y, -y_max, y_max));
     axes.push_back(std::make_shared<util::RegularAxis>(n_t, 0, constants::TWO_PI));
```

Comment:
If this initializer list works as I hope, it might be cleaner:
```suggestion
    auto const axes = util::Axes {std::make_shared<util::RegularAxis>(n_x, -x_max, x_max),
        std::make_shared<util::RegularAxis>(n_y, -y_max, y_max),
        std::make_shared<util::RegularAxis>(n_t, 0, constants::TWO_PI)};
```

This should work by using the `std::initializer_list<T>` constructor of a `std::vector<T>`:
https://github.com/glotzerlab/freud/blob/3dad11885dcfebfa79fe2575f4e8b556ab3caba2/cpp/util/Histogram.h#L277
https://en.cppreference.com/w/cpp/container/vector/vector

Key benefit: the vector is initialized with the correct size and elements, and thus can be declared `const`.

You might make the same change for other instances of this pattern everywhere that `Axes` is used.